ynh-dev

```diff
@@ -365,11 +383,14 @@ with open(setting_file) as f:
 EOF
                 fi
 
-                local DISTRO="$(lsb_release -s -c)"
-
                 cd "$DEV_PATH"
+                systemctl start yunohost-api
+                # Inject container ip in .env file
+                # Used by vite to expose itself on network and proxy api requests.
+                echo "VITE_IP=$(hostname -I | tr ' ' '\n' | grep "\.")" > "$DEV_PATH/.env"
+                create_sym_link "$DEV_PATH/.env" "$CACHE_PATH/.env"
```

Comment:
The `.env` file seemedto never contain the correct IP. I hope sourcing it at the very last moment fixes it.